src/Shortcode/Shortcode.php

```diff
@@ -225,18 +225,18 @@ private function autoEmbedYouTube($text)
                 (?:https?://)?      # Optional scheme. Either http or https.
                 (?:[0-9A-Z-]+\.)?   # Optional subdomain.
                 (?:                 # Group host alternatives.
-                  youtu\.be/       # Either youtu.be (trailing slash required),
+                  youtu\.be/        # Either youtu.be (trailing slash required),
                 | youtube\.com      # or youtube.com followed by
                   \S*               # Allow anything up to VIDEO_ID,
-                  [^\w\-\s]         # but char before ID is non-ID char.
+                  [^\w\\-\s]        # but char before ID is non-ID char.
```

Comment:
Was this intentional?

Reply:
More or less. Rector changed that using the `RegexDashEscapeRector` rule.
Interestingly enough PHP's regex dialect does not make much of a difference there, PHPStorm's internal syntax check gave me similar responses in the past, ultimately resulting in me changed quite a few of the other regexes. 

I verified youtube emebs still work as intended via `/test/shortcode.php`.